scripts/build/debug/300-gdb.sh

```diff
@@ -285,7 +286,6 @@ do_gdb_backend()
     fi
 
     if [ "${static}" = "y" ]; then
-        cflags+=" -static"
         ldflags+=" -static"
```

Comment:
Does changing this to `-all-static` work?

Reply:
I tried that at first, but this causes the `gdb` `./configure` step to fail, because it expects the toolchain to be able to build an executable without `libtool`, and in that case `-all-static` is not recognized.